namespaces/live-1.cloud-platform.service.justice.gov.uk/claim-criminal-injuries-compensation-uat/resources/rds.tf

```diff
@@ -0,0 +1,36 @@
+variable "cluster_name" {}
+variable "cluster_state_bucket" {}
+
+module "rds" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-rds-instance?ref=4.0"
```

Comment:
This module was updated a couple of weeks ago. Please use `ref=4.1`

Reply:
I have updated rds-instance ref to 4.1

Reply:
I have updated rds instance ref to 4.1